assets/js/src/notices/notice.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { FC, ReactNode } from 'react';
 import ReactDOM from 'react-dom';
-import PropTypes from 'prop-types';
 import MailPoet from 'mailpoet';
 
-const Notice = ({
+type Props = {
+  type: 'success' | 'info' | 'warning' | 'error';
+  children: ReactNode;
+  scroll?: boolean;
+  closable?: boolean;
+  renderInPlace?: boolean;
+  onDisplay?: () => void;
+  onClose?: () => void;
+  timeout?: number | false;
+};
+
+const Notice: FC<Props> = ({
```

Comment:
Same question if we need to use `FC` here or if there's other syntax. Also... is it really nicer to unwrap the properties to such a large number of variables? Kind of thinking if `props.onClose`... isn't more convenient.

Reply:
The props were spread like that before this change. But yeah I would prefer to use `props.*` in this case. Do you think I should make the change in this PR?